vortexasdk/endpoints/canal_transit_result.py

```diff
@@ -0,0 +1,76 @@
+import functools
+import os
+from multiprocessing.pool import Pool
+from typing import List
+
+import pandas as pd
+
+from vortexasdk.api import CanalTransitRecord
+from vortexasdk.api.entity_flattening import (
+    convert_to_flat_dict,
+)
+from vortexasdk.api.search_result import Result
+from vortexasdk.logger import get_logger
+from vortexasdk.result_conversions import create_dataframe, create_list
+
+logger = get_logger(__name__)
+
+DEFAULT_COLUMNS = [
+    "vessel_id",
+    "vessel_name",
+    "vessel_imo",
+    "vessel_class.0.label",
+    "vessel_cubic_capacity",
+    "vessel_dead_weight",
+    "booked",
+    "canal",
+    "direction",
+    "lock",
+    "queue_arrival_time",
+    "canal_entry_time",
+    "cargoes.0.product.label",
+    "cargoes.0.quantity_barrels",
+    "cargoes.0.quantity_tonnes",
+    "origin.0.label",
+    "destination.0.label",
+]
```

Comment:
Once we merge publish typo fix one more col will be added - `quantity_cubic_metres`